src/group/groupFragments.js

```diff
@@ -27,3 +27,15 @@ export const groupMembers = `
     }
   }
 `
+
+export const accountMembership = `
```

Comment:
Thought: `account` is a concept we don't have yet. Should we keep using `User` only? Do you see any problem using two words (user, account) to refer to the same thing (user)?

Reply:
@cacarrara I do think there is an account notion (at least from the web client side). So, I see 3 concepts referencing people from the web client:
Account: Anything related to the current logged in user, or the user performing the actions
Members: People related to a group (this can be inderectly members of the landscape too, through the default group)
Users: I think this will apply to user profile probabily when we get the requirement to access other people profiles

So, at least right now I don't see a confusion, when `account` is used it is linked to the current logged in user, and things mentioning users are refercing something to users in general, let me know where are you seeing this confusion